### PR TITLE
Add Team page

### PR DESCRIPTION
**Pull request by jonasschnelli**
Refs #137

### CODE REVIEW

**Comment by jonasschnelli** [comment]
Not sure if we have missed someone or if the ordering is wrong. Feedback welcome.

**Comment by laanwj** [comment]
ACK!

File: _data/contributors.yml [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+    name: Jonas Schnelli
+    github: jonasschnelli
+    irc: jonasschnelli
+    twitter: jonasschnelli
+    avatar: https://avatars1.githubusercontent.com/u/178464?v=3&s=460
+    bio:
+    role: GUI maintainer
+contributors:
+  -
+    name: Pieter Wullie
```
Comment (paveljanik) [inline _data/contributors.yml]: Who is it? ;-)

**Comment by CodeShark** [comment]
ACK

File: _includes/pages/team/_list.html [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<ul class="team">
+{% for person in people %}
+{% assign avatar=person.avatar %}
+{% unless person.avatar contains 'https://' %}{% capture avatar %}/images/team/{{person.avatar}}{% endcapture %}{% endunless %}
+<li>
+    <a href="https://github.com/{{ person.github }}"><img src="{{ avatar }}" class="bio-photo" width="150" height="150" alt="{{ name }}"></a>
```
Comment (btcdrak) [inline _includes/pages/team/_list.html]: `alt="{{ name }}">` should be `alt="{{ name }}" />` which fixes the CI error.

File: _data/contributors.yml (hunk 1 above)
Comment (btcdrak) [inline _data/contributors.yml]: s/Wullie/Wuille

**Comment by btcdrak** [comment]
ACK

File: _posts/en/pages/2016-03-14-team.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+### Maintainers
+      
+Project maintainers are responsible for merging patches from contributors. They perform a janitorial role merging patches that the team agrees should be merged. They also act as a final check to ensure that patches are safe and in line with the project goals. The maintainers' role is by agreement of project contributors.  
+
+{% include pages/team/_maintainers.html %}
+
+### Contributors
+
+Everyone who contributes to the Bitcoin Core project is considered a contributor. The list below are some of the current major contributors to the codebase. There are many others who contribute more causally. A list of code contributors for the last year can be found on [Github][recent-contributors] as well in the [historical release notes][1] for each Bitcoin Core software release.
```
Comment (dcousens) [inline _posts/en/pages/2016-03-14-team.md]: s/causally/casually

**Comment by jonasschnelli** [comment]
I'll also add Daniel Cousens.

**Comment by btcdrak** [comment]
Merged with nits addressed.

**Pull request closed without merging**